.github/workflows/bat.yml

```diff
@@ -61,6 +61,5 @@ jobs:
             (test -f test-results/modelcoverage.xml) || (echo "File modelcoverage.xml does not exist" && exit 1)
             (test -f test-results/pdf-results.pdf) || (echo "File pdf-results.pdf does not exist" && exit 1)
             grep -q "runner.addPlugin(FailOnWarningsPlugin())" console.log
-            grep -q "parallel pool" console.log
```

Comment:
`grep -q "runInParallel" console.log` ?

Reply:
Yeah good call. It looks like this should work in both cases: it finds PCT or doesn't. I'd like to add PCT back into the setup-matlab stage when the gcp issue is fixed